simulation/executor/init_functions.go

```diff
@@ -1,4 +1,4 @@
-package simtypes
```

Comment:
Why did we move this file?

Recommend not moving it, since these are type definitions

Reply:
In conjunction with the comment below, if we want to use our local definition of Config, we need to use our local definition of both RandomAccountFn and AppStateFn. Because executor folder imports simtypes, we cant have simtypes import executor (as far as I have learned) due to a cyclical import error. This is why I moved this file to executor so I can still call our local implementation of both RandomAccountFn and AppStateFn. If the below comment I made about Config is moot then this can easily be switched back.